examples/organization-configs/README.md

```diff
@@ -0,0 +1,34 @@
+# Organization config examples 
+
+You can find here examples of organization configs that provide defaults
+for all of the organization's tasks. It is useful to put there some options
+your organization always uses (like `judge_type`) or enable checks disabled
+by default.
+
+Organization configs are stored in the `configs` subdirectory of the `pisek`
+settings folder. The folder is automatically detected at the topmost level
+of the git repository root. You can override the location of `pisek` settings
+folder with either `--pisek-dir` or `$PISEK_DIRECTORY`.
+
+```
+my-organization
+├── .git
+└── pisek
+    └── configs
+        ├── cms-base-v1
+        ├── cms-batch-v1
+        └── cms-communication-v1
+```
+
+It might be a good idea to add versions to your organization configs, as if you want to make
+breaking change, it is clearer that you should use the newest version.
```

Comment:
I'm not sure what you mean by "it is clearer that you should use the newest version".

```suggestion
It might be a good idea to add versions to your organization configs, as if you want to make
a breaking change, you can simply start using a new version.
```